pkg/policy/policy_service.go

```diff
@@ -127,6 +139,48 @@ func (p *policyService) ManualAttachArbitraryPolicy(roleName string, policyArns
 	return cmd
 }
 
+func (p *policyService) AutoDetachArbitraryPolicy(roleName string, policyArns []string,
+	accountID, orgID string) (string, error) {
+	output := ""
+	for _, policyArn := range policyArns {
+		err := p.AWSClient.DetachRolePolicy(policyArn, roleName)
+		if err != nil {
+			if awserr.IsNoSuchEntityException(err) {
+				output = output + fmt.Sprintf("Currently policy %s is not attached to role %s\n",
+					policyArn, roleName)
+			} else {
+				return output, fmt.Errorf("Failed to detach policy %s from role %s: %s",
```

Comment:
"Failed to detach policy '%s' from role '%s': %s",

Reply:
updated